tests/gradients/core/test_hadamard_gradient.py

```diff
@@ -440,6 +456,7 @@ def test_prob_expectation_values(self, tol):
             qml.CNOT(wires=[0, 1])
             qml.expval(qml.PauliZ(0))
             qml.probs(wires=[0, 1])
+            qml.probs()
```

Comment:
Should we also add a test case with: `qml.probs(op=op)` ?

Reply:
Sure, added :)

Reply:
Awesome!

Reply:
I was confused on where you added it because I thought it would be added on the `QuantumScript` in `test_prob_expectation_values` but I see you implemented it in a separate cost function. Works as well!